packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,344 @@
+  // This ensures semantics are distinct from appearance.
+  font: inherit;
+  // margin applied to top of heading, informed by current margin size
+  margin-block: 1.5em 0;
+}
+
+p {
+  // ❓margin applied to top of text, informed by Chrome UA style - do we want to codify it?
+  margin-block: 1em 0;
+
+  // ❓taken from ./_base.scss ln 329 - do we want to add this to the reset?
+  // CMS-WDS/application @extends this style for their reset:
+  // https://github.cms.gov/CMS-WDS/application/blob/6de8f3ac19e86b7653f0102dcfff7a79241dcb33/app/styles/base/index.scss#L14
+  &:first-child {
+    margin-block-start: 0;
+  }
+}
+
+// These tags will naturally use 700 weight as bold
+// but because some themes have 600 weight, I'm using
+// the CSS variable to adjust bold based on theme
+b,
+strong {
+  font-weight: var(--font-weight-bold);
+}
+
+a {
+  color: var(--link__color);
+  // 📢 We may want to revisit tokens for `text-decoration-style`
+  // ❓Do we have an instance where the underline doesn't match the text color?
+  text-decoration-color: currentColor;
+  text-decoration-style: solid;
+  text-decoration-thickness: var(--link__text-decoration-thickness);
+  text-underline-offset: var(--link__text-decoration-offset);
+
+  // Recommended order of user action pseudo-classes:
+  // https://css-tricks.com/snippets/css/link-pseudo-classes-in-order/
+
+  &:visited {
+    color: var(--link__color--visited);
+  }
+
+  &:hover {
+    color: var(--link__color--hover);
+    text-decoration-thickness: var(--link__text-decoration-thickness--hover);
+  }
+
+  &:focus,
+  &:focus:visited {
+    background-color: var(--color-focus-light);
+    box-shadow: none;
+    color: currentColor;
+    outline: 3px solid var(--color-focus-dark);
+    outline-offset: 1px;
+  }
+
+  &:active {
+    color: var(--link__color--active);
+  }
+}
+
+// ❓ How do we want to style <hr> tags? I'm using best guess here.
+hr {
+  border-block-end: 0;
+  border-block-start: 1px solid currentColor;
+  margin: 0;
+}
+
+/* Lists */
+
+// Relying on `flex` to set `gap` spacing size on li elements
+// `gap` ensures even spacing and doesn't interfere with layout if list given a `row` direction
+ol,
+ul {
+  // display: flex;
+  // flex-direction: column;
+  // gap: .5em;
+  margin-block: 1em 0;
+  padding-inline-start: 2em;
+
+  // Tying unstyled list styles to a required a11y attribute ensures proper markup structure
+  &[role='list'] {
+    list-style: none;
+    padding: 0;
+  }
+}
+
+// 📢 We don't have examples of nested lists in the design system
+// ❓ How do we handle li `::mark` and nested padding for this pattern?
+// If we choose to support nesting lists, need to remove `gap` solution in favor of li `margin-block-start`
+// li elements are `display: list-item` and cannot receive `flex` or `grid` display without removing `::mark`
+li {
+  margin-block: 0.5em 0;
+}
+
+/* Form elements */
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
```

Comment:
I think this font declaration is good to include in the reset code, but I kind of think it's unnecessary to reset all the form elements since we have classes specifically for those things. Or we could possibly restrict the declarations to things like margin. Otherwise, what's the point of those other classes?

Reply:
The bounds of the design system styles are shifting a lot by this reset, and we may have to move the line several times before we strike the right balance.

Reply:
These elements have crazy user agent styles without being reset, and within the DS use same font styles as the page itself (family, size, line-height, etc.), so generically setting the font styles for these elements like this enables us to remove unnecessary CSS from the classes redeclaring the same things over and over. 

For instance, [these lines of CSS](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/styles/components/_Button.scss#L13-L19) could be removed from our Button styles and [these](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/styles/components/_TextField.scss#L51-L53) from our TextFields.

Reply:
I'm on board with this line that I highlighted and other font stuff and margins, but I just wasn't sure if we needed stuff like `border-radius` and `background-color` because they are maybe redundant with the classes that we use 🤷‍♂️

Reply:
Oh, yeah maybe for the inputs themselves I can remove a lot of the non-font, layout styles. I'm good with that.